tests/fixtures/addon/defaults-travis/package.json

```diff
@@ -62,7 +64,11 @@
     "loader.js": "^4.7.0",
     "prettier": "^2.3.0",
     "qunit": "^2.15.0",
```

Comment:
Someone unrelated, but is it weird that this package.json file lacks dependencies like webpack and tracked-built-ins?

Reply:
A couple of the other package.json files below are missing one or both of those packages, inconsistently. Not something to fix in this PR, but maybe worth looking into if it wasn't intentional.

Reply:
Yeah, I've noticed that's been an inconsistency for a long time, not sure what's going on. But shouldn't affect my PR.